README.md

```diff
@@ -9,13 +9,14 @@ A node manager helps to manage the host kernel configuration of the [Harvester](
 - `v0.1.x` branch is used for Harvester v1.2.x release.
 - `v0.2.x` branch is used for Harvester v1.3.x release.
 - `v0.3.x` branch is used for Harvester v1.4.x release.
+- `v1.5` branch is used for Harvester v1.5.x release.
 
 ## Manifests and Deploying
 The `./manifests` folder contains useful YAML manifests to use for deploying and developing the Harvester node manager.
 This simply YAML deployment creates a Daemonset using the `rancher/harvester-node-manager` container.
 
 ## License
-Copyright (c) 2025 [Rancher Labs, Inc.](http://rancher.com)
+Copyright (c) 2025 [SUSE, LLC.](https://www.suse.com/)
```

Comment:
Perhaps not related to this PR, should we also do the same thing in scripts/boilerplate.go.txt?

Reply:
@innobead How should we handle such things? How important is it to update the copyright?